### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by akumarujon**
### Documentation changes for several types.
List of types:
- [x] ChosenInlineResult
- [x] Game
- [x] CallbackQuery
- [x] LoginUrl
- [x] Poll
- [x] ReplyQuote
- [x] InlineQuery
- [x] InlineQueryResultButton
- [x] Giveaway
- [x] GiveawayParameters
- [x] MessageInteractions
- [x] MessageReference
- [x] PriceTag
- [x] InactiveChat
- [x] MessageReactionCount
- [x] ReactionCount
- [x] MessageReactions
- [ ] MaskPosition
- [ ] NetworkStatistics
- [ ] NetworkStatisticsEntry
- [ ] RestrictionReason
- [ ] Story
- [ ] StoryInteractions
- [ ] StoryReference

### CODE REVIEW

File: types/2_chosen_inline_result.ts [hunk 1]
```diff
@@ -6,10 +6,15 @@ import { constructUser, User } from "./1_user.ts";
 
 /** A chosen inline result. */
 export interface ChosenInlineResult {
+  /** The unique identifier for the result that was chosen. */
   resultId: string;
+  /** The user that chose the result. */
```
Comment (roj1512) [inline types/2_chosen_inline_result.ts:11]:
```suggestion
  /** The user who chose the result. */
```

File: types/2_chosen_inline_result.ts [hunk 2]
```diff
@@ -6,10 +6,15 @@ import { constructUser, User } from "./1_user.ts";
...
   from: User;
+  /** Sender location, only for bots that require user location. */
```
Comment (roj1512) [inline types/2_chosen_inline_result.ts:13]:
```suggestion
  /** The location of the user who chose the result. */
```

File: types/1_inline_query_result_button.ts [hunk 3]
```diff
@@ -2,7 +2,10 @@ import { MiniAppInfo } from "./0_mini_app_info.ts";
 
 /** A button to be shown along with the results of an inline query. */
 export interface InlineQueryResultButton {
+  /** Label text on the button. */
   text: string;
+  /** Description of the Web App that will be launched when the user presses the button. */
   miniApp?: MiniAppInfo;
+  /**  */
```
Comment (akumarujon) [inline types/1_inline_query_result_button.ts:9]: help wanted

File: types/2_inline_query.ts [hunk 4]
```diff
@@ -6,10 +6,15 @@ import { constructUser, User } from "./1_user.ts";
...
   id: string;
+  /** Sender of this query/ */
   from: User;
+  /** Text of the query (up to 256 characters). */
   query: string;
+  /** Offset of the results to be returned. */
   offset: string;
+  /** Type of the chat from which the inline query was sent. */
   chatType?: "sender" | "private" | "group" | "supergroup" | "channel";
   location?: Location;
```
Comment (akumarujon) [inline types/2_inline_query.ts:19]: and here

File: types/1_inline_query_result_button.ts (hunk 3 above)
Comment (roj1512) [reply types/1_inline_query_result_button.ts:9]:
```suggestion
  /** Deep linking parameter for the /start message. */
```

File: types/2_inline_query.ts (hunk 4 above)
Comment (roj1512) [inline types/2_inline_query.ts:19]:
```suggestion
  chatType?: "sender" | "private" | "group" | "supergroup" | "channel";
  /** The location of the user who made the inline query. */
  location?: Location;
```

File: types/1_user.ts [hunk 5]
```diff
@@ -8,31 +8,31 @@ export interface User {
   id: number;
   /** Identifier of color that can be displayed instead of the user's photo. */
   color: number;
-  /** True, if this user is a bot */
+  /** Whether this is a bot’s chat. */
```
Comment (roj1512) [inline types/1_user.ts:11]:
```suggestion
  /** Whether the user is a bot. */
```

File: types/1_user.ts [hunk 6]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
+  /** Whether the user is a Telegram Premium user. */
   isPremium: boolean;
-  /** True, if the user is verified */
+  /** Whether the user has been verified. */
   isVerified: boolean;
-  /** True, if the user is an official support user */
+  /** Whether the user is official support. */
   isSupport: boolean;
-  /** True, if this user added the bot to the attachment menu */
+  /** Whether the user added the bot to the attachment menu. */
```
Comment (roj1512) [inline types/1_user.ts:35]:
```suggestion
  /** Whether the user is a bot that has been added to the attachment menu by the current user. */
```

File: types/1_user.ts [hunk 7]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
   isBot: boolean;
-  /** User's or bot's first name */
+  /** User's or bot's first name. */
   firstName: string;
-  /** User's or bot's last name */
+  /** User's or bot's last name. */
   lastName?: string;
-  /** User's or bot's username */
+  /** User's or bot's username. */
   username?: string;
```
[the remark this answers is not in the document]
Comment (akumarujon) [reply types/1_user.ts]: Is this okay that `ChatPPrivate` doesn't have `photo`?

**Comment by akumarujon** [comment]
in commit https://github.com/MTKruto/MTKruto/pull/193/commits/9fbcd567eca026f78f8e2e4ae3f4d21e98ef9f1c some changes are made in:

ReactionCount
MessageReactionCount
MessageReactions

File: types/1_giveaway.ts [hunk 8]
```diff
@@ -3,8 +3,11 @@ import { constructGiveawayParameters, GiveawayParameters } from "./0_giveaway_pa
 
 /** A giveaway. */
 export interface Giveaway {
+  /** Giveaway parameters. */
```
Comment (roj1512) [inline types/1_giveaway.ts:6]:
```suggestion
```

File: types/1_inline_query_result_button.ts [hunk 9]
```diff
@@ -2,7 +2,10 @@ import { MiniAppInfo } from "./0_mini_app_info.ts";
 
 /** A button to be shown along with the results of an inline query. */
 export interface InlineQueryResultButton {
+  /** Label text on the button. */
   text: string;
+  /** Description of the Web App that will be launched when the user presses the button. */
```
Comment (roj1512) [inline types/1_inline_query_result_button.ts:7]: We call them mini apps, not web apps.

File: types/1_poll.ts [hunk 10]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
 
 /** A poll. */
 export interface Poll {
+  /** Unique poll identifier. */
   id: string;
+  /** Poll question, 1-300 characters. */
```
Comment (roj1512) [inline types/1_poll.ts:10]:
```suggestion
  /** The poll's question. */
```

File: types/1_poll.ts [hunk 11]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
 
 /** A poll. */
 export interface Poll {
+  /** Unique poll identifier. */
```
Comment (roj1512) [inline types/1_poll.ts:8]:
```suggestion
  /** The poll's unique identifier. */
```

File: types/1_poll.ts [hunk 12]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   id: string;
+  /** Poll question, 1-300 characters. */
   question: string;
+  /** List of poll options. */
```
Comment (roj1512) [inline types/1_poll.ts:12]:
```suggestion
  /** The poll's options. */
```

File: types/1_poll.ts [hunk 13]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   options: PollOption[];
+  /** Total number of users that voted in the poll. */
```
Comment (roj1512) [inline types/1_poll.ts:14]:
```suggestion
  /** The number of users who have participated in the poll. */
```

File: types/1_poll.ts [hunk 14]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   totalVoterCount: number;
+  /** True, if the poll is closed. */
   isClosed: boolean;
+  /** True, if the poll is anonymous. */
```
Comment (roj1512) [inline types/1_poll.ts:18]: "Whether ...", not "True, if ..."

File: types/1_poll.ts [hunk 15]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   isAnonymous: boolean;
+  /** Poll type, currently can be “regular” or “quiz”. */
   type: "regular" | "quiz";
+  /** True, if the poll allows multiple answers. */
```
Comment (roj1512) [inline types/1_poll.ts:22]: "Whether ...", not "True, if ..."

File: types/1_poll.ts [hunk 16]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   question: string;
+  /** List of poll options. */
   options: PollOption[];
+  /** Total number of users that voted in the poll. */
   totalVoterCount: number;
+  /** True, if the poll is closed. */
   isClosed: boolean;
+  /** True, if the poll is anonymous. */
   isAnonymous: boolean;
+  /** Poll type, currently can be “regular” or “quiz”. */
```
Comment (roj1512) [inline types/1_poll.ts:20]:
```suggestion
  /** The type of the poll. */
```

File: types/1_poll.ts [hunk 17]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   type: "regular" | "quiz";
+  /** True, if the poll allows multiple answers. */
   allowMultipleAnswers?: boolean;
+  /** 0-based identifier of the correct answer option. */
```
Comment (roj1512) [inline types/1_poll.ts:24]:
```suggestion
  /** Index of the correct option. */
```

File: types/1_poll.ts [hunk 18]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   correctOptionIndex?: number;
+  /** Text that is shown when a user chooses an incorrect answer or taps on the lamp icon in a quiz-style poll, 0-200 characters. */
```
Comment (roj1512) [inline types/1_poll.ts:26]:
```suggestion
  /** A text that is shown to the user when the poll is answered. */
```

File: types/1_poll.ts [hunk 19]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   explanation?: string;
+  /** Special entities like usernames, URLs, bot commands, etc. that appear in the explanation. */
```
Comment (roj1512) [inline types/1_poll.ts:28]:
```suggestion
  /** The explanation's entities. */
```

File: types/1_poll.ts [hunk 20]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   explanationEntities?: MessageEntity[];
+  /** Amount of time in seconds the poll will be active after creation. */
```
Comment (roj1512) [inline types/1_poll.ts:30]:
```suggestion
  /** Duration of the poll in seconds. */
```

File: types/1_poll.ts [hunk 21]
```diff
@@ -5,18 +5,31 @@ import { constructPollOption, PollOption } from "./0_poll_option.ts";
...
   openPeriod?: number;
+  /** Point in time (Unix timestamp) when the poll will be automatically closed. */
```
Comment (roj1512) [inline types/1_poll.ts:32]:
```suggestion
  /** The time in which the poll will be closed. */
```

File: types/1_reaction_count.ts [hunk 22]
```diff
@@ -3,7 +3,9 @@ import { constructReaction, Reaction } from "./0_reaction.ts";
 
 /** The count of a specific reaction. */
 export interface ReactionCount {
+  /** Type of the reaction. */
```
Comment (roj1512) [inline types/1_reaction_count.ts:6]:
```suggestion
  /** The type of the reaction. */
```

File: types/1_user.ts [hunk 23]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
+  /** The first name of the user. */
   firstName: string;
-  /** User's or bot's last name */
+  /** The last name of the user. */
   lastName?: string;
-  /** User's or bot's username */
+  /** The user’s main username. */
   username?: string;
-  /** Additional usernames */
+  /** Additional usernames. */
```
Comment (roj1512) [inline types/1_user.ts:19]:
```suggestion
  /** The user's additional usernames. */
```

File: types/1_user.ts [hunk 24]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
   also?: string[];
-  /** The user's photo. */
+  /** The photo of the user. */
```
Comment (roj1512) [inline types/1_user.ts:21]:
```suggestion
  /** The user's profile photo. */
```

File: types/1_user.ts [hunk 25]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
   photo?: ChatPhotoUser;
-  /** [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag) of the user's language */
+  /** [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag) of the user's language. */
```
Comment (roj1512) [inline types/1_user.ts:23]:
```suggestion
  /** The user's [IETF language tag](https://en.wikipedia.org/wiki/IETF_language_tag). */
```

File: types/1_user.ts [hunk 26]
```diff
@@ -8,31 +8,31 @@ export interface User {
...
   languageCode?: string;
-  /** True, if the user is a scam user */
+  /** Whether the user has been identified as scam. */
   isScam: boolean;
-  /** True, if this user was reported by many users as a fake or scam user: be careful when interacting with them. */
+  /** Whether the user has been identified as an impersonator. */
   isFake: boolean;
-  /** True, if this user is a Telegram Premium user */
+  /** Whether the user is a Telegram Premium user. */
```
Comment (roj1512) [inline types/1_user.ts:29]:
```suggestion
  /** Whether the user is subscribed to Telegram Premium. */
```

File: types/1_giveaway.ts [hunk 27]
```diff
@@ -3,8 +3,11 @@ import { constructGiveawayParameters, GiveawayParameters } from "./0_giveaway_pa
...
   parameters: GiveawayParameters;
+  /** Number of users which will receive Telegram Premium subscription gift codes. */
   winnerCount: number;
+  /** Number of months the Telegram Premium subscription will be active after code activation. */
```
Comment (roj1512) [inline types/1_giveaway.ts:10]:
```suggestion
  /** The duration of each Telegram Premium subscription that is to be given away in months. */
```

File: types/1_giveaway.ts [hunk 28]
```diff
@@ -3,8 +3,11 @@ import { constructGiveawayParameters, GiveawayParameters } from "./0_giveaway_pa
 
 /** A giveaway. */
 export interface Giveaway {
+  /** Giveaway parameters. */
   parameters: GiveawayParameters;
+  /** Number of users which will receive Telegram Premium subscription gift codes. */
```
Comment (roj1512) [inline types/1_giveaway.ts:8]:
```suggestion
  /** The quantity of the Telegram Premium subscriptions that will be given away.. */
```

File: types/0_price_tag.ts
Comment (roj1512) [inline types/0_price_tag.ts:1]: Please don’t document this for now.

File: types/0_message_reference.ts
Comment (roj1512) [inline types/0_message_reference.ts:1]: Please don’t document this for now.

File: types/0_login_url.ts [hunk 29]
```diff
@@ -1,7 +1,11 @@
 /** A login URL. */
 export interface LoginUrl {
+  /** An HTTPS URL to be opened with user authorization data added to the query string when the button is pressed. If the user refuses to provide authorization data, the original URL without information about the user will be opened. */
   url: string;
+  /** New text of the button in forwarded messages. */
   forwardText?: string;
+  /** Username of a bot, which will be used for user authorization. */
```
Comment (roj1512) [inline types/0_login_url.ts:7]:
```suggestion
  /** The username of the bot that performs the authorization. */
```

File: types/0_login_url.ts [hunk 30]
```diff
@@ -1,7 +1,11 @@
 /** A login URL. */
 export interface LoginUrl {
+  /** An HTTPS URL to be opened with user authorization data added to the query string when the button is pressed. If the user refuses to provide authorization data, the original URL without information about the user will be opened. */
```
Comment (roj1512) [inline types/0_login_url.ts:3]:
```suggestion
```

File: types/0_giveaway_parameters.ts [hunk 31]
```diff
@@ -3,10 +3,15 @@ import { peerToChatId, types } from "../2_tl.ts";
 
 /** Giveaway parameters. */
 export interface GiveawayParameters {
+  /** Identifier of the supergroup or channel chat, which will be automatically boosted by the winners of the giveaway for duration of the Premium subscription. */
```
Comment (roj1512) [inline types/0_giveaway_parameters.ts:6]:
```suggestion
  /** The identifier of the chat that will be boosted for the duration of the given away Premium subscriptions. */
```

File: types/0_giveaway_parameters.ts [hunk 32]
```diff
@@ -3,10 +3,15 @@ import { peerToChatId, types } from "../2_tl.ts";
...
   boostedChatId: number;
+  /** Identifiers of other supergroup or channel chats that must be subscribed by the users to be eligible for the giveaway. */
```
Comment (roj1512) [inline types/0_giveaway_parameters.ts:8]:
```suggestion
  /** The identifiers of additional chats that the user must subscribe to in order to be eligible for the prizes. */
```

File: types/0_giveaway_parameters.ts [hunk 33]
```diff
@@ -3,10 +3,15 @@ import { peerToChatId, types } from "../2_tl.ts";
...
   additionalChatIds: number[];
+  /** Point in time (Unix timestamp) when the giveaway is expected to be performed. */
```
Comment (roj1512) [inline types/0_giveaway_parameters.ts:10]:
```suggestion
  /** The point of time in which the winners will be chosen. */
```

File: types/0_giveaway_parameters.ts [hunk 34]
```diff
@@ -3,10 +3,15 @@ import { peerToChatId, types } from "../2_tl.ts";
...
   winnerSelectionDate: Date;
+  /** Whether the new members of the chats will be eligible for the giveaway. */
```
Comment (roj1512) [inline types/0_giveaway_parameters.ts:12]:
```suggestion
  /** Whether only new members of the chats will be eligible for the prizes. */
```

File: types/0_giveaway_parameters.ts [hunk 35]
```diff
@@ -3,10 +3,15 @@ import { peerToChatId, types } from "../2_tl.ts";
...
   onlyNewMembers: boolean;
+  /** The list of two-letter ISO 3166-1 alpha-2 codes of countries, users from which will be eligible for the giveaway. */
```
Comment (roj1512) [inline types/0_giveaway_parameters.ts:14]:
```suggestion
  /** A list of countries that are eligible for the prizes. */
```

File: types/1_reply_quote.ts [hunk 36]
```diff
@@ -3,8 +3,11 @@ import { constructMessageEntity, MessageEntity } from "./0_message_entity.ts";
 
 /** A reference to a specific part of a message that is being replied to. */
 export interface ReplyQuote {
+  /** Offset of the replied message from the start of the chat session, in bytes. */  
   offset: number;
+  /** Text of the quoted part of a message that is replied to by the given message. */
   text: string;
+  /** Special entities that appear in the quote. */
```
Comment (roj1512) [inline types/1_reply_quote.ts:10]:
```suggestion
  /** The entities of the quoted text. */
```

File: types/1_reply_quote.ts [hunk 37]
```diff
@@ -3,8 +3,11 @@ import { constructMessageEntity, MessageEntity } from "./0_message_entity.ts";
 
 /** A reference to a specific part of a message that is being replied to. */
 export interface ReplyQuote {
+  /** Offset of the replied message from the start of the chat session, in bytes. */
```
Comment (roj1512) [inline types/1_reply_quote.ts:6]:
```suggestion
  /** The byte offset of the quoted text. */  
```

File: types/1_reply_quote.ts [hunk 38]
```diff
@@ -3,8 +3,11 @@ import { constructMessageEntity, MessageEntity } from "./0_message_entity.ts";
 
 /** A reference to a specific part of a message that is being replied to. */
 export interface ReplyQuote {
+  /** Offset of the replied message from the start of the chat session, in bytes. */  
   offset: number;
+  /** Text of the quoted part of a message that is replied to by the given message. */
```
Comment (roj1512) [inline types/1_reply_quote.ts:8]:
```suggestion
  /** The quoted text. */
```

File: types/2_chosen_inline_result.ts [hunk 39]
```diff
@@ -6,10 +6,15 @@ import { constructUser, User } from "./1_user.ts";
 
 /** A chosen inline result. */
 export interface ChosenInlineResult {
+  /** The unique identifier for the result that was chosen. */
```
Comment (roj1512) [inline types/2_chosen_inline_result.ts:9]:
```suggestion
  /** The identifier of the inline result that was chosen. */
```

File: types/2_game.ts [hunk 40]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
 
 /** A game. */
 export interface Game {
+  /** Title of the game. */
```
Comment (roj1512) [inline types/2_game.ts:10]:
```suggestion
  /** The title of the game. */
```

File: types/2_game.ts [hunk 41]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
...
   title: string;
+  /** Description of the game. */
```
Comment (roj1512) [inline types/2_game.ts:12]:
```suggestion
  /** The description of the game. */
```

File: types/2_game.ts [hunk 42]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
...
   description: string;
+  /** Photo that will be displayed in the game message in chats. */
```
Comment (roj1512) [inline types/2_game.ts:14]:
```suggestion
  /** A photo that is displayed when the game is shared. */
```

File: types/2_inactive_chat.ts [hunk 43]
```diff
@@ -3,7 +3,9 @@ import { enums, types } from "../2_tl.ts";
 import { ChatP, constructChatP } from "./1_chat_p.ts";
 
 export interface InactiveChat {
+  /** Date of the last activity in Unix time. */
```
Comment (roj1512) [inline types/2_inactive_chat.ts:6]:
```suggestion
  /** The point of time in which the chat was last active. */
```

File: types/2_inactive_chat.ts [hunk 44]
```diff
@@ -3,7 +3,9 @@ import { enums, types } from "../2_tl.ts";
...
   lastActivity: Date;
+  /** The chat. */
```
Comment (roj1512) [inline types/2_inactive_chat.ts:8]:
```suggestion
  /** The chat that has been marked as inactive. */
```

File: types/2_inactive_chat.ts [hunk 45]
```diff
@@ -3,7 +3,9 @@ import { enums, types } from "../2_tl.ts";
 import { ChatP, constructChatP } from "./1_chat_p.ts";
 
 export interface InactiveChat {
```
Comment (roj1512) [inline types/2_inactive_chat.ts:5]:
```suggestion

/** An inactive chat. */
export interface InactiveChat {
```

File: types/2_game.ts [hunk 46]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
...
   photo: Photo;
+  /** Brief description of the game or high scores included in the game message. */
```
Comment (roj1512) [inline types/2_game.ts:16]:
```suggestion
  /** A text that is displayed when the game is shared. */
```

File: types/2_game.ts [hunk 47]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
...
   text?: string;
+  /** Special entities that appear in text, such as usernames, URLs, bot commands, etc. */
```
Comment (roj1512) [inline types/2_game.ts:18]:
```suggestion
  /** The text's entities. */
```

File: types/2_game.ts [hunk 48]
```diff
@@ -7,11 +7,17 @@ import { constructPhoto, Photo } from "./1_photo.ts";
...
   textEntities?: MessageEntity[];
+  /** Animation that will be displayed in the game message in chats. */
```
Comment (roj1512) [inline types/2_game.ts:20]:
```suggestion
  /** An animation that is displayed when the game is shared. */
```

File: types/2_inline_query.ts [hunk 49]
```diff
@@ -6,11 +6,17 @@ import { constructUser, User } from "./1_user.ts";
 
 /** An incoming inline query. */
 export interface InlineQuery {
+  /** Unique identifier for this query. */
```
Comment (roj1512) [inline types/2_inline_query.ts:9]:
```suggestion
  /** The identifier of the inline query.. */
```

File: types/2_inline_query.ts [hunk 50]
```diff
@@ -6,11 +6,17 @@ import { constructUser, User } from "./1_user.ts";
...
   id: string;
+  /** Sender of this query/ */
```
Comment (roj1512) [inline types/2_inline_query.ts:11]:
```suggestion
  /** The user who made the query. */
```

File: types/2_inline_query.ts [hunk 51]
```diff
@@ -6,11 +6,17 @@ import { constructUser, User } from "./1_user.ts";
...
   from: User;
+  /** Text of the query (up to 256 characters). */
```
Comment (roj1512) [inline types/2_inline_query.ts:13]:
```suggestion
  /** The query that was made. */
```

File: types/2_inline_query.ts [hunk 52]
```diff
@@ -6,11 +6,17 @@ import { constructUser, User } from "./1_user.ts";
...
   query: string;
+  /** Offset of the results to be returned. */
```
Comment (roj1512) [inline types/2_inline_query.ts:15]:
```suggestion
  /** The offset parameter that was passed to the last [answerInlineQuery](/methods/answerInlineQuery) call. */
```

File: types/2_inline_query.ts [hunk 53]
```diff
@@ -6,11 +6,17 @@ import { constructUser, User } from "./1_user.ts";
...
   offset: string;
+  /** Type of the chat from which the inline query was sent. */
```
Comment (roj1512) [inline types/2_inline_query.ts:17]:
```suggestion
  /** The type of the chat from which the inline query was made. */
```

File: types/2_message_interactions.ts [hunk 54]
```diff
@@ -2,9 +2,14 @@ import { MessageReaction } from "./1_message_reaction.ts";
 
 /** The interactions made with a message. */
 export interface MessageInteractions {
+  /** ID of the chat where the message was sent. */
```
Comment (roj1512) [inline types/2_message_interactions.ts:5]:
```suggestion
  /** The identifier of the message's chat. */
```

File: types/2_message_interactions.ts [hunk 55]
```diff
@@ -2,9 +2,14 @@ import { MessageReaction } from "./1_message_reaction.ts";
...
   chatId: number;
+  /** Unique identifier of the message. */
```
Comment (roj1512) [inline types/2_message_interactions.ts:7]:
```suggestion
  /** The message's identifier. */
```

File: types/2_message_interactions.ts [hunk 56]
```diff
@@ -2,9 +2,14 @@ import { MessageReaction } from "./1_message_reaction.ts";
...
   messageId: number;
+  /** The reactions made to the message. */
   reactions: MessageReaction[];
+  /** Number of times the message was viewed. */
   views: number;
+  /** Number of times the message was forwarded. */
```
Comment (roj1512) [inline types/2_message_interactions.ts:13]:
```suggestion
  /** The number of times the message was viewed. */
  views: number;
  /** The number of times the message was forwarded. */
```

File: types/2_message_reaction_count.ts [hunk 57]
```diff
@@ -6,9 +6,13 @@ import { constructReactionCount, ReactionCount } from "./1_reaction_count.ts";
 
 /** Information on the reactions made to a channel post. */
 export interface MessageReactionCount {
+  /** The chat containing the message. */
   chat: ChatP;
+  /** Unique message identifier inside the chat. */
```
Comment (roj1512) [inline types/2_message_reaction_count.ts:11]:
```suggestion
  /** The message's identifier. */
```

File: types/2_message_reaction_count.ts [hunk 58]
```diff
@@ -6,9 +6,13 @@ import { constructReactionCount, ReactionCount } from "./1_reaction_count.ts";
...
   messageId: number;
+  /** Date of the change in Unix time. */
```
Comment (roj1512) [inline types/2_message_reaction_count.ts:13]:
```suggestion
  /** This reaction state's point of time. */
```

File: types/2_message_reaction_count.ts [hunk 59]
```diff
@@ -6,9 +6,13 @@ import { constructReactionCount, ReactionCount } from "./1_reaction_count.ts";
...
   date: Date;
+  /** List of reactions that are present on the message. */
```
Comment (roj1512) [inline types/2_message_reaction_count.ts:15]:
```suggestion
  /** The reactions made to the post. */
```

File: types/2_message_reactions.ts [hunk 60]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
 
 /** The reactions of a user to a messages in a group. */
 export interface MessageReactions {
+  /** The chat containing the message the user reacted to. */
   chat: ChatP;
+  /** Unique identifier of the message inside the chat. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:12]:
```suggestion
  /** The message's identifier. */
```

File: types/2_message_reactions.ts [hunk 61]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
...
   messageId: number;
+  /** The user that changed the reaction, if the user isn't anonymous. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:14]:
```suggestion
  /** The user that changed their reactions to the message. Unset if done on behalf of a chat. */
```

File: types/2_message_reactions.ts [hunk 62]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
...
   user?: User;
+  /** The chat on behalf of which the reaction was changed, if the user is anonymous. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:16]:
```suggestion
  /** The chat that changed its reactions to the message. Unset if done on behalf of a user. */
```

File: types/2_message_reactions.ts [hunk 63]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
...
   actorChat?: ChatP;
+  /** Date of the change in Unix time. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:18]:
```suggestion
  /** The point of time in which the change was made. */
```

File: types/2_message_reactions.ts [hunk 64]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
...
   date: Date;
+  /** Previous list of reaction types that were set by the user. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:20]:
```suggestion
  /** The previous reactions. */
```

File: types/2_message_reactions.ts [hunk 65]
```diff
@@ -7,12 +7,19 @@ import { constructUser, User } from "./1_user.ts";
...
   oldReactions: Reaction[];
+  /** New list of reaction types that have been set by the user. */
```
Comment (roj1512) [inline types/2_message_reactions.ts:22]:
```suggestion
  /** The current reactions. */
```

File: types/5_callback_query.ts [hunk 66]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
 
 /** A received callback query. */
 export interface CallbackQuery {
+  /** Unique identifier for this query. */
```
Comment (roj1512) [inline types/5_callback_query.ts:9]:
```suggestion
  /** The identifier of the callback query. */
```

File: types/5_callback_query.ts [hunk 67]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   id: string;
+  /** Sender of the query. */
```
Comment (roj1512) [inline types/5_callback_query.ts:11]:
```suggestion
  /** The user who made the callback query. */
```

File: types/5_callback_query.ts [hunk 68]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   from: User;
+  /** Message sent by the bot with the callback button that originated the query. */
```
Comment (roj1512) [inline types/5_callback_query.ts:13]:
```suggestion
  /** The message from which the callback query was made. Unset if made from an inline result message. */
```

File: types/5_callback_query.ts [hunk 69]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   message?: Message;
+  /** Identifier of the message sent via the bot in inline mode, that originated the query. */
```
Comment (roj1512) [inline types/5_callback_query.ts:15]:
```suggestion
  /** The identifier of the inline result message from which the callback query was made. Unset if made from a message not originating from an inline query result. */
```

File: types/5_callback_query.ts [hunk 70]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   inlineMessageId?: string;
+  /** Global identifier, uniquely corresponding to the chat to which the message with the callback button was sent. */
```
Comment (roj1512) [inline types/5_callback_query.ts:17]:
```suggestion
  /** A special identifier for the chat in which the callback was made from. Useful for inline result messages. */
```

File: types/5_callback_query.ts [hunk 71]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   chatInstance: string;
+  /** Data associated with the callback button. Be aware that the message originated the query can contain no callback buttons with this data. */
   data?: string;
+  /** Short name of a Game to be returned, serves as the unique identifier for the game. */
```
Comment (roj1512) [inline types/5_callback_query.ts:21]:
```suggestion
  /** The short name of the game to be returned. */
```

File: types/5_callback_query.ts [hunk 72]
```diff
@@ -6,12 +6,19 @@ import { Message, MessageGetter } from "./4_message.ts";
...
   id: string;
+  /** Sender of the query. */
   from: User;
+  /** Message sent by the bot with the callback button that originated the query. */
   message?: Message;
+  /** Identifier of the message sent via the bot in inline mode, that originated the query. */
   inlineMessageId?: string;
+  /** Global identifier, uniquely corresponding to the chat to which the message with the callback button was sent. */
   chatInstance: string;
+  /** Data associated with the callback button. Be aware that the message originated the query can contain no callback buttons with this data. */
```
Comment (roj1512) [inline types/5_callback_query.ts:19]:
```suggestion
  /** The data associated with the button that was used to make the callback query. */
```

File: test.ts
Comment (roj1512) [inline test.ts:1]: Remove this file.

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much! If you had the time, the remaining types can be updated in a different PR.

**Pull request merged**